source/server/listener_impl.h

```diff
@@ -24,6 +24,18 @@
 namespace Envoy {
 namespace Server {
 
+class ListenerMessageUtil {
+public:
+  /**
+   * This function is used by determine if the lhs config could be updated to rhs config by
+   * upgrading the ListenerConfig at ConnectionHandler. It depends on the implementation of
+   * ConnectionHandler.
```

Comment:
What does "It depends on the implementation of ConnectionHandler" mean? Clarify or remove?

Reply:
The idea is, if in the future connection handler could handle udp/api active listener in place update, we should relax the filterChainOnlyChange().
Another possibility is that we add a painless field in v3:listener proto message, it would better that we can execute that listener update as filterChainOnlyChange().